src/support/components/PayGSupportOverlay.tsx

```diff
@@ -1,31 +1,141 @@
+    '3 - Standard',
+    '4 - Request',
+  ]
+
+  const handleInputChange = (e: ChangeEvent<HTMLTextAreaElement>) => {
+    setInput(e.target.value)
+  }
+  const handleSubmit = (): void => {
+    // submit support form
+  }
+
+  const handleChangeSeverity = (severity): void => {
+    setSeverity(severity)
+  }
+
+  const severityTip = (): JSX.Element => {
+    const tooltipContent = (
+      <div>
+        Please refer to our severity levels in our
+        <SafeBlankLink href="https://www.influxdata.com/legal/support-policy">
+          {' '}
+          support policy{' '}
+        </SafeBlankLink>
+        website
+      </div>
+    )
+    return (
+      <QuestionMarkTooltip
+        diameter={14}
+        tooltipContents={tooltipContent}
+        tooltipStyle={{fontSize: '13px'}}
+        style={{marginLeft: '420px'}}
+      />
+    )
+  }
+
   return (
-    <Overlay.Container maxWidth={500}>
+    <Overlay.Container maxWidth={550}>
       <Overlay.Header
         testID="payg-support-overlay-header"
         title="Contact Support"
         onDismiss={onClose}
       />
-
+      <p className="status-page-text">
+        <span>
+          {' '}
+          <Icon glyph={IconFont.Info_New} />{' '}
+        </span>
+        Check our{' '}
+        <SafeBlankLink href="https://status.influxdata.com">
+          status page
+        </SafeBlankLink>{' '}
+        to see if there is an outage impacting your region.
+      </p>
       <ErrorBoundary>
-        <Overlay.Body></Overlay.Body>
+        <Overlay.Body>
+          <Form
+            onSubmit={handleSubmit}
+            action="https://influxdata--full.my.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8"
+            method={Method.Post}
+          >
+            <Form.Element
+              label="Severity"
+              required={true}
+              labelAddOn={severityTip}
+            >
+              <SelectDropdown
+                options={severityLevel}
+                selectedOption={severity}
+                onSelect={handleChangeSeverity}
+                indicator={DropdownItemType.None}
+              />
+            </Form.Element>
+            <Form.Element label="Description" required={true}>
+              <TextArea
+                rows={10}
+                testID="support-description--textarea"
+                name="description"
+                maxLength={2500}
+                value={input}
+                onChange={handleInputChange}
+              />
+            </Form.Element>
+            <Overlay.Footer>
+              <Button
+                text="Cancel"
+                color={ComponentColor.Tertiary}
+                onClick={onClose}
+                type={ButtonType.Button}
+                testID="payg-contact-support--cancel"
+              />
+              <Button
+                text="Submit"
+                color={ComponentColor.Success}
+                type={ButtonType.Submit}
+                testID="payg-contact-support--submit"
```

Comment:
maybe consider adding a `disabled` property here so that when there's no text in the textfield, this button is disabled

Reply:
great idea. Better yet, I think we need a validation function that makes sure severity level and text area are properly filled in. for instance, for text area, user has to write at least 20 characters (at least one sentence) to be able to submit a form. I'll create a separate issue for this.

Reply:
ah nevermind, I ended up adding the code to this pr.